rmw_connext_cpp/CMakeLists.txt

```diff
@@ -56,7 +59,9 @@ register_rmw_implementation(
 )
 
 add_library(rmw_connext_cpp SHARED src/functions.cpp)
+target_link_libraries(rmw_connext_cpp ${rcutils_LIBRARIES})
```

Comment:
This line is redundant with `rcutils` being listed two lines below with `ament_target_dependencies`.